src/styles/helpers/color/getBrightness.js

```diff
@@ -0,0 +1,17 @@
+import isValidColor from './isValidColor';
+import decomposeColor from './decomposeColor';
+import hslToRgb from './hslToRgb';
+
+/* http://www.w3.org/TR/AERT#color-contrast */
+
+/* eslint-disable no-param-reassign */
+export default function getBrightness(color) {
```

Comment:
we must add tests to this helper, but can be done in another PR

Reply:
ok